src/storhaug

```diff
@@ -368,6 +394,51 @@ setup_create_resources()
     pcs -f ${cibfile} constraint location ganesha_state-clone rule resource-discovery=exclusive score=0 role eq storage
 
     pcs cluster cib-push ${cibfile} || storlog "ERR" "Failed to create filesystem resources."
+}
+
+# nothing needed for now
+setup_create_resources_shared_rados_kv()
+{
+    :
+}
+
+setup_create_resources_shared()
+{
+    if [[ ${HA_NFS_TYPE} == "glusterfs" ]]; then
+        setup_create_resources_shared_glusterfs
+    elif [[ ${HA_NFS_TYPE} == "rados_kv" ]]; then
+        setup_create_resources_shared_rados_kv
+    fi
+}
+
+setup_create_resources_nfs()
+{
+    local cibfile="$1"
+
+    setup_create_resources_shared ${cibfile}
```

Comment:
@jarrpa The function is moved into setup_create_resources_nfs(), we still need it, because there are different state volume type for 'glusterfs' and 'rados_kv'